### PR TITLE
content: jordan spotlight

### PR DESCRIPTION
**Pull request by alexnm**

### CODE REVIEW

File: src/posts/spotlights/jordan.md [hunk 1]
```diff
@@ -10,18 +10,105 @@ mapsLink: https://www.google.com/maps/place/Jordan/@31.1675526,36.1454843,7.94z/
...
+  - Warm clothes for the desert night
+  - Comfortable shoes
+  - A light backpack for carrying food and water for the day
 ---
 
-# Petra and Wadi Rum
+## Understand
 
-## Wow
+Jordan is a small country in the Middle East, often considered one of the most stable from the political and economic perspective. The country attracts more than 1 million visitors every year, most of whom are coming specially for the rock city of Petra, one of the [seven new wonders of the world](https://en.wikipedia.org/wiki/New7Wonders_of_the_World).
```
Comment (melaniam) [inline src/posts/spotlights/jordan.md:22]: "the most stable from the economic and political perspective"
"most of whom are coming especially"

File: src/posts/spotlights/jordan.md [hunk 2]
```diff
@@ -10,18 +10,105 @@ mapsLink: https://www.google.com/maps/place/Jordan/@31.1675526,36.1454843,7.94z/
...
 
-- one
-- two
-- three
+Petra and Wadi Rum are usually visited together due to their proximity, but of course you can choose a different itinerary.
 
-### Nice!
+## How to organize your trip
+
+We visited Jordan during [a longer trip that also covered Israel](/travel/a-taste-of-the-middle-east/). But of course you can visit Jordan separately since you can easily spend somewhere around a week touring the entire country.
```
Comment (melaniam) [inline src/posts/spotlights/jordan.md:35]: you can easily spend around a week touring the entire country.

File: src/posts/spotlights/jordan.md [hunk 3]
```diff
@@ -10,18 +10,105 @@ mapsLink: https://www.google.com/maps/place/Jordan/@31.1675526,36.1454843,7.94z/
...
+
+Since we only had a few days and getting in and out of Jordan from Israel is not that straight-forward (it's not as hard as you think though), we decided to book a trip with a Israeli company. It turned out to be the best decision as we had an amazing experience with [Abraham Tours](https://www.abrahamtours.com/) and we definitely recommend them to everyone.
```
Comment (melaniam) [inline src/posts/spotlights/jordan.md:37]: an Israeli company

File: src/posts/spotlights/jordan.md [hunk 4]
```diff
@@ -10,18 +10,105 @@ mapsLink: https://www.google.com/maps/place/Jordan/@31.1675526,36.1454843,7.94z/
...
+
+A lot of travel agencies in Israel offer trips to Petra and Jordan in general and if you're short on time, it is definitely your best option as they take care of everything for you.
+
+Petra and Wadi Rum are located in the south part of Jordan, so the best starting point is in Eilat, in the south of Israel, by the Red Sea. You can also book trips directly from Jerusalem or Tel Aviv, but you spend a lot of time on the road and it's not worth it. Plus, Eilat is a great destination by itself and there are plenty of things to do there, if you choose to stay a few days.
```
Comment (melaniam) [inline src/posts/spotlights/jordan.md:41]: are located in the southern part of Jordan, so the best starting point is Eilat, ...
if you choose to stay for a few days.

File: src/posts/spotlights/jordan.md [hunk 5]
```diff
@@ -10,18 +10,105 @@ mapsLink: https://www.google.com/maps/place/Jordan/@31.1675526,36.1454843,7.94z/
...
+
+So in the middle of our stay in Eilat, we booked [this 2 day trip](https://www.abrahamtours.com/tours/petra-and-wadi-rum-2-day-from-eilat/) to Jordan. Bear in mind that while the trip itself seems expensive when you consider the period, Jordan in general is a very expensive country.
```
Comment (melaniam) [inline src/posts/spotlights/jordan.md:43]: Keep in mind

File: src/posts/spotlights/jordan.md [hunk 6]
```diff
@@ -10,18 +10,105 @@ mapsLink: https://www.google.com/maps/place/Jordan/@31.1675526,36.1454843,7.94z/
...
+
+We were 12 people in the group, so it was a great experience overall. Someone from the travel agency organized our departure from Eilat and we were driven to the border which is just outside the city. Here, there was another person waiting on the Jordanian side (you cross the actual border by foot) that guided us to our small bus. There we met our guide, Mahmoud, who was with us for the entire duration of the trip.
```
Comment (melaniam) [inline src/posts/spotlights/jordan.md:45]: We were 12 people in the group, and it was a great experience overall. Someone from the travel agency organized our departure from Eilat and we were driven to the border, which is just outside the city. We crossed the actual border on foot, and there was another person waiting for us on the Jordanian side. He helped us with the legal formalities and guided us to our small bus. There we met our guide, Mahmoud, who was with us for the entire duration of the trip.

File: src/posts/spotlights/jordan.md [hunk 7]
```diff
@@ -10,18 +10,105 @@ mapsLink: https://www.google.com/maps/place/Jordan/@31.1675526,36.1454843,7.94z/
...
+<figure>
+  <img src="/img/jordan/group-at-petra.jpeg" alt="Group of people leaving Petra">
+  <figcaption>Our international group.</figcaption>
+</figure>
+
+We've been to a lot of countries and we had a lot of guides in the past few years but we have to say Mahmoud ranks somewhere at the very top of the list. He was extremly funny, highly knowledgable and overall created a wonderful experience for everyone in the group.
+
+## Activities
+
+We crossed the border in the morning and headed to Acqba, the town on the Jordanian side of the Red Sea. Here we had breakfast at a local restaurant and then jumped back in the bus for the two hour drive to Wadi Rum.
```
Comment (melaniam) [inline src/posts/spotlights/jordan.md:56]: After crossing the border, we were headed to Acqba, the city on the Jordanian side ..

File: src/posts/spotlights/jordan.md [hunk 8]
```diff
@@ -10,18 +10,105 @@ mapsLink: https://www.google.com/maps/place/Jordan/@31.1675526,36.1454843,7.94z/
...
+
+The fun part began as soon as we entered the Wadi Rum National Park and exchanged our bus with two open 4x4 pickup trucks for our desert tour.
+
+### Jeep tour in Wadi Rum
+
+Touring the Wadi Rum desert is an sureal experience. You feel small as you climb on to a sand dune to get a 360 degree view of the surrounding area. There's something about the emptiness of the desert that cannot be described, it has to be felt in person.
```
Comment (melaniam) [inline src/posts/spotlights/jordan.md:62]: a surreal experience
on top of a sand dune

File: src/posts/spotlights/jordan.md [hunk 9]
```diff
@@ -10,18 +10,105 @@ mapsLink: https://www.google.com/maps/place/Jordan/@31.1675526,36.1454843,7.94z/
...
+There are dozens of camps around the desert that welcome guests from all over the world. We had a room ready at [Mohammed Mutlak Camp](http://www.booking.com/Share-5V1h7N) part of our trip, but you can also book on your own as most of the properties are on booking.com.
+
+<figure>
+  <img src="/img/jordan/wadi-rum-2.jpg" alt="">
+  <figcaption>Overlooking the camp.</figcaption>
+</figure>
+
+A room in a camp here is actually a wool tent with a hard floor, built and decorated in a traditional bedouin style. There's a big bed in the middle of the room with plenty of blankets for the cold night. There is also a power outlet and a single light source, but don't worry, you won't be staying in your room unless you're sleeping. The camp has a separate bigger building where you have shared toilets, running water and showers with hot water. The shared bathrooms are really clean and well maintained.
+
+Other than that, the camp area has an open fire where they serve tee non-stop and two bigger tents for eating and hanging around with the rest of the guests. One of the tents is enclosed and warmed. Dinner and breakfast were served by the staff of the camp and the food was delicious.
```
Comment (melaniam) [inline src/posts/spotlights/jordan.md:87]: they serve tea
with the other guests

File: src/posts/spotlights/jordan.md [hunk 10]
```diff
@@ -10,18 +10,105 @@ mapsLink: https://www.google.com/maps/place/Jordan/@31.1675526,36.1454843,7.94z/
...
+  <figcaption>Sunset view.</figcaption>
+</figure>
+
+The best part is that there is no internet connection or phone signal, so you actually get to spend your time with the other guests. We had a great time getting to know our trip companions better and meeting other people from all over the world during the evening.
+
+Later during the night, you should definitely go outside and enjoy the starry night sky. There are few places on Earth where the light polution is so low.
+
+### Petra
+
+The second day of the trip was decicated entirely to Petra. All in all, we were at the site for about 6 hours, enough time to wander around and catch the best shots for your photo album. There are numerous hikes that you can take at Petra, some which require multiple hours to complete, so you can easily spend 1-2 full days there.
```
Comment (melaniam) [inline src/posts/spotlights/jordan.md:100]: dedicated
for about 6 hours. That's enough time to wander around and catch the best shots for your photo album. However, there are numerous hikes that you can take at Petra, some of which require multiple hours to complete, so you can easily spend 1-2 full days there if you have the time.

File: src/posts/spotlights/jordan.md [hunk 11]
```diff
@@ -10,18 +10,105 @@ mapsLink: https://www.google.com/maps/place/Jordan/@31.1675526,36.1454843,7.94z/
...
+
+The city is famous for its rock carvings and one particulary
```
Comment (melaniam) [inline src/posts/spotlights/jordan.md:102]: ???

File: src/posts/spotlights/jordan.md [hunk 12]
```diff
@@ -10,18 +10,105 @@ mapsLink: https://www.google.com/maps/place/Jordan/@31.1675526,36.1454843,7.94z/
...
+
+## Travel tips
+
+Overall, Jordan is a tourist-friendly and safe country that you can definitely visit on your own.
+
+### Money
+
+The local currency is the Jordanian Dinar (JOD), one of the strongest currencies in the world actually (1 JOD = 1.28 EUR as of 2020). You can exchange USD or EUR in Acqba or you can use a local ATM to get some money. Keep in mind that you need at least 40 JOD for the border fee when you exit Jordan and you can only pay that amount in their currency.
```
Comment (melaniam) [inline src/posts/spotlights/jordan.md:110]: you can only pay that amount in their currency, cash.

File: src/posts/spotlights/jordan.md (hunk 11 above)
Comment (alexnm) [reply src/posts/spotlights/jordan.md:102]: WIP

**Pull request merged**